src/garage/replay_buffer/sac_replay_buffer.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+
+
+class SACReplayBuffer:
```

Comment:
how does this differ from any of our other replay buffers?

Reply:
I the goal here just to store one sample at a time? Can you just add a method to `PathBuffer` instead that inserts samples outside of a trajectory?

Reply:
This slipped into the PR because I was testing primitives to see if they were broken. I can remove this.